qiita_pet/support_files/doc/source/gettingstartedguide/index.rst

```diff
@@ -139,6 +139,10 @@ Upload Sample Information
 
   * Drag your sample info file into the box to upload your files
   * Your uploaded samples will appear under the "Files" section
+  * You can currently upload biom tables and .fastq , .fastq.gz, .txt, .tsv, .sff, .fasta, .fna, and .qual files
```

Comment:
Can you replace this by something like: `You can currently upload biom tables and .fastq, .fastq.gz, ...., among other; for a full list check the upload page of any study` ... my concern is that this will get outdated when we allow other filetypes.

Reply:
Same question here as I asked earlier. Should I make it:

* You can currently upload biom tables, .fastq files, .fastq.gz files, among others.
  For a full list check the upload page of any study.

    * *Note that '.zip' files can not be processed.*
    * *Note that '.fasta' and '.fna' files require '.qual' files for submission.*
    * *Note that '.txt. files must be tab separated and require the extension .txt.*